geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -238,7 +238,8 @@ impl GeyserPluginManager {
             return Err(jsonrpc_core::Error {
                 code: ErrorCode::InvalidRequest,
                 message: format!(
-                    "There already exists a plugin named {} loaded, while reloading {name}. Did not load requested plugin",
+                    "There already exists a plugin named {} loaded, while reloading {name}. Did \
```

Comment:
This was before your formatting, but I don't really like that one is passed as argument and the other is embedded. Maybe move name out from `{}`?